src/pages/Home/components/ClubsTab.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState, useEffect, ChangeEvent } from 'react';
+import { useRecoilValue } from 'recoil';
+import { toast } from 'react-toastify';
+import { SearchIcon } from '@heroicons/react/solid';
+import OwnedTokenGrid from './OwnedTokenGrid';
+import { useTokenRegistry } from '../../../common/components/Solana/TokenRegistry';
+import GenerateTokenOwnedList from '../../../common/services/Solana/GetData/GenerateTokenOwnedList';
+import { userPublicKeyAtom } from '../../../recoil/userInfo';
+
+import { ItokenOwned } from '../../../common/types';
+
+export default function ClubsTab(): JSX.Element {
+  const userPublicKey = useRecoilValue(userPublicKeyAtom);
+  const tokenRegistry = useTokenRegistry();
+  const [tokenOwnedList, setTokenOwnedList] = useState<ItokenOwned[]>([]);
+  const [tokenOwnedSearchedList, setTokenOwnedSearchedList] = useState<
```

Comment:
ownedSearch sounds strange. Why don't we replace by tokenOwnedSearchResults

Reply:
okk